src/Orangebeard.Client/OrangebeardV2Client.cs

```diff
@@ -0,0 +1,257 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Orangebeard.Client.Entities;
+using Orangebeard.Client.OrangebeardProperties;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orangebeard.Client
+{
+    public class OrangebeardV2Client : AbstractClient
+    {
+        //private readonly ILogger<OrangebeardV2Client> LOGGER;
+
+        private bool connectionWithOrangebeardIsValid = false;
+        private readonly OrangebeardConfiguration config;
+
+        private OrangebeardV2Client()
+        {
+            //var loggerFactory = (ILoggerFactory)new LoggerFactory();
+            //LOGGER = loggerFactory.CreateLogger<OrangebeardV2Client>();
+        }
+
+        private void Init(string endpoint, Guid uuid)
+        {
+            InitializeHttpClient(new Uri(endpoint), uuid.ToString(), null);
+
+            ServicePointManager.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true; //TODO?- Not sure if this is still necessary.
```

Comment:
Did this have to do with the .Net version discrepancy?

Reply:
No, I'm just not sure if this is required only by ReportPortal, or if it is something we need in general. But I'm guessing the latter; there's no ReportPortal specific code in there.